crypto/crypto.go

```diff
@@ -43,11 +46,19 @@ func GenerateKey() (*ecdsa.PrivateKey, error) {
 	return ecdsa.GenerateKey(S256(), rand.Reader)
 }
 
+// ToECDSAPub create ecdsa.PublicKey object by byte array. If pub length equals 64, then need inserts one byte in front
 func ToECDSAPub(pub []byte) *ecdsa.PublicKey {
 	if len(pub) == 0 {
 		return nil
 	}
-	x, y := elliptic.Unmarshal(S256(), pub)
+	if len(pub) == 65 {
```

Comment:
The previous code came from ETH. 1) How about adding necessary comment? 2) Is it necessary to check the "pub" length?

Reply:
1.comment is above func definition. Maybe it is not enough
2.yes, I also doubt. But consider there is no check before,  make no check